docs/platform/reference/eol-for-major-versions.rst

```diff
@@ -0,0 +1,167 @@
+EOL for major versions of Aiven Services
+========================================
+
+End of Life (EOL) refers to the deadline after which affected Aiven
+services are retired and will be no longer supported nor maintained.
+
+Since August 2020, Aiven aims to follow the EOL schedule set by the
+original authors and maintainers of the open source software, aka
+upstream projects. Once the upstream project retires a specific version,
+they do not receive security updates and critical bug fixes anymore by
+the maintainers.
+
+Continued use of outdated services means that they no longer offer our
+customers the level of protection their business needs. Therefore, by
+following the upstream project's EOL schedule, we ensure that Aiven
+services are always running on supported versions of the open source
+software.
+
+**Version numbering**
+~~~~~~~~~~~~~~~~~~~~~
+
+Aiven services inherit the upstream project’s software versioning
+scheme. Depending on the service, a major version can be either a single
+digit (e.g. PostgreSQL® 12) or ``major.minor`` (e.g. Kafka® 2.4). The
+exact version of the service is visible in the Aiven console once the
+service is up and running.
+
+Aiven for Elasticsearch
+-----------------------
+
+Aiven for Elasticsearch major versions will reach EOL on the same date
+as the upstream open source project's EOL .
+
+.. container:: intercom-interblocks-table-container
+
+   =========== ============= ==================
+   **Version** **Aiven EOL** **Upstream EOL**
+   2.x         2020-10-30    2018-02-28
+   5.x         2020-10-30    2019-03-11
+   6.x         2020-11-20    2020-11-20
+   7.x         2022-03-23    2022-05-11 (v7.10)
+   =========== ============= ==================
+
+Aiven for PostgreSQL®
+---------------------
+
+Aiven for PostgreSQL® major versions will reach EOL on the same date as
+the upstream open source project's EOL .
+
+.. container:: intercom-interblocks-table-container
+
+   +-------------+---------------+------------------+------------------+
+   | **Version** | **Aiven EOL** | **Availability   | **Availability   |
```

Comment:
Can we remove the old ones we don't offer anymore?

Reply:
I would advise against removing the old ones as this also serves as history of the versions we have supported and ended. It is a common product practise to retain the EOL info. As an example, here is the page maintained by Elasticsearch: https://www.elastic.co/support/eol